cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,31 @@
 /// <reference types="cypress" />
 
+Cypress.on('uncaught:exception', (_err, runnable) => {
+  return false;
+});
+
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
+    cy.visit('https://react-redux.realworld.io');
+
+    cy.get(':nth-child(2) > .nav-link')
+      .click();
+    cy.get(':nth-child(1) > .form-control')
+      .type('yatut.coffee+1@gmail.com');
+    cy.get(':nth-child(2) > .form-control')
+      .type('P@s$w0rd');
+    cy.get('.btn')
+      .click();
+
+    cy.get(':nth-child(4) > .nav-link')
+      .contains('YaA');
+    cy.get('.container > .nav > :nth-child(2) > .nav-link')
```

Comment:
this and the code below is not needed. We do need to check header and clicking on settings link